### DISCUSSION TITLE
Install on Raspberry Pi OS Lite

### DISCUSSION THREAD

**Discussion by joerg-wille** [rimago/docker-kodi#13] [Q&A]
I have installed:
Raspberry Pi OS Lite
Release date: May 3rd 2023
System: 64-bit
Kernel version: 6.1
Debian version: 11 (bullseye)

I have installed current docker and portainer.
What packages do I need to install to run Kodi in a container?

**Comment by rimago** [comment]
This is the same setup I have in use. If you want to use it with pulseaudio you should set this up, but this is not necessary. You can simply try it with the docker compose file from [Usage](https://github.com/rimago/rpi-kodi#Usage).
